pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -1152,6 +1177,58 @@ def test_tdi_div_tdlike_scalar_with_nat(self, two_hours, box_with_array):
         result = rng / two_hours
         tm.assert_equal(result, expected)
 
+        result = two_hours / rng
+        expected = 1 / expected
+        tm.assert_equal(result, expected)
+
+    def test_td64arr_div_td64_ndarray(self, box_with_array):
+        # GH#22631
+        rng = TimedeltaIndex(['1 days', pd.NaT, '2 days'])
+        expected = pd.Float64Index([12, np.nan, 24])
```

Comment:
Nit: this variable name is misleading given that you overwrite it two lines later.

Reply:
You're right, but this pattern is repeated all over these tests.

Reply:
Fair enough.  Don't 100% agree with the pattern, but we can always clean it up in a follow-up.